react/wbooks/.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_USER_BASE_URL=https://wbooks-api-stage.herokuapp.com/api/v1
```

Comment:
i would ignore this in .gitignore, but for this training its ok, remember to never add a .env into the repository

Reply:
This is the .env.development which is not in gitignore. I thought you use this file like an env example.
I should add this to the gitignore? Which is the purpose of this file? The env var is obtained from the .env file, not from here (I tested this)